contracts/managers/SocialTradingManager.sol

```diff
@@ -342,6 +416,24 @@ contract SocialTradingManager is
         );
     }
 
+    /*
+     * Validate new entry fee.
+     *
+     * @param _entryFee      New entry fee in a scaled decimal value
+     *                          (e.g. 100% = 1e18, 1% = 1e16)
+     */
+    function validateNewEntryFee(
+        uint256 _entryFee
+    )
+        internal
+        view
+    {
+        require(
+            _entryFee <= maxEntryFee,
+            "SocialTradingManager.validateNewEntryFee: Passed entry fee must not exceed maxEntryFee."
+        );
```

Comment:
Do we want to validate that it's a multiple of a basis point? If so, you can import `ScaleValidations` in set-protocol-contracts

Reply:
Nm, that would be redundant to the setEntryFee on the RBSet